### DISCUSSION TITLE
Make assets possible to embed on a webpage

### DISCUSSION THREAD

**Discussion by TheAnachronism** [alangrainger/immich-public-proxy#32] [Feature Requests]
It would be really useful if assets made public with the share link could be embedded on a webpage.
The current share link will open a web-interface, but not direct access to the asset behind it.

I'm not sure if this is possible the way immich makes share links available, but I hope this might be possible.

**Comment by alangrainger** [comment]
This is already how things work if you share either a single image, or link directly to an image file:

For example, this image from the demo album which I will embed here:

```
![](https://immich-demo.note.sx/share/photo/ffSw63qnIYMtpmg0RNvOui0Dpio7BbxsObjvH8YZaobIjIAzl5n7zTX5d6EDHdOYEvo/b4169f32-3259-40d6-b3a8-bf6a2f10e63c/thumbnail)
```

![](https://immich-demo.note.sx/share/photo/ffSw63qnIYMtpmg0RNvOui0Dpio7BbxsObjvH8YZaobIjIAzl5n7zTX5d6EDHdOYEvo/b4169f32-3259-40d6-b3a8-bf6a2f10e63c/thumbnail)

If you want to embed a whole album, you can do that with an iframe.

If you share a single image, the share link itself is embeddable, like this:

```
![](https://immich-demo.note.sx/share/tx-1XemhcEdMwLKhP9uTv9V6duXs-CBKCalFOH00ElOQYjY7PYf8yi7jMZy09zPXXm4)
```

![](https://immich-demo.note.sx/share/tx-1XemhcEdMwLKhP9uTv9V6duXs-CBKCalFOH00ElOQYjY7PYf8yi7jMZy09zPXXm4)

Does that answer the request?

**Comment by alangrainger** [comment]
I'll close this pending a reply from @TheAnachronism . We can re-open if the above does not solve the issue.
